create_aio_app/template/{{cookiecutter.project_name}}/Makefile

```diff
@@ -121,3 +122,7 @@ adev: {%- if cookiecutter.use_postgres == 'y' or cookiecutter.use_redis == 'y' %
 wait_resources:
 	python3 -m $(PROJECT_NAME).utils.wait_script
 {%- endif %}
+
+## Create profile of application.
+profile:
+	@docker exec -it $(PROJECT_NAME) py-spy record -d $(TIME) -o $(PROJECT_NAME)_profile.svg --pid 7
```

Comment:
We generate pretty output based on makefile so let's move profile under upgrade command for better order

Reply:
```
py-spy works from the command line and takes either the PID of the program you want to sample
```

i'm not sure but i guess so that in docker-compose the  main process has pid #1. Are u sure that 7 is correct?

Reply:
is process id of python the same each time?

Reply:
Yes, they seem to remain the same.

Reply:
Okay, please do fix with cookiecutter.json and will made it

Reply:
Sure thing, i think we can make changes if someone sees something else.